tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateSidewaysView.java

```diff
@@ -141,9 +141,11 @@ static int getFirstScript(UnicodeSet exemplars) {
     static Comparator<Object> UCA;
 
     static {
-        RuleBasedCollator UCA2 = (RuleBasedCollator) Collator.getInstance(ULocale.ROOT);
+        RuleBasedCollator UCA2 =
+                (RuleBasedCollator) Collator.getInstance(ULocale.ROOT); // freeze below
         UCA2.setNumericCollation(true);
         UCA2.setStrength(Collator.IDENTICAL);
+        UCA2 = (RuleBasedCollator) UCA2.freeze();
```

Comment:
This collator could also be added to CollatorHelper, numeric and strength IDENTICAL